frontend/package.json

```diff
@@ -2,7 +2,9 @@
   "name": "aml-frontend",
   "version": "0.1.0",
   "private": true,
-  "homepage": "/",
+  "homepage": "app.amsterdammusiclab.nl",
+  "description": "The React Frontend for the MUSCLE platform",
+  "license": "MIT",
```

Comment:
We probably don't need any information but the description in the frontend `package.json`. The version here might also be removed. But that might again raise all kinds of warnings, I suppose? Oh yes, `private: true` can be `false`, both here and in the main `package.json`. We don't have any plans to release (parts of) MUSCLE via npm, but also no need to block release.